lesson07/assignment/my-app-name/src/views/Accordion.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div id="accordion">
+    <div class="card" v-for="(card, index) in cards" v-bind:key="card">
+      <div class="card-header" :id="'heading'+ (index + 1)">
+        <h5 class="mb-0">
+          <button class="btn btn-link" data-toggle="collapse" :data-target="'#collapse'+(index + 1)" aria-expanded="false" :aria-controls="'collapse'+(index + 1)">
```

Comment:
Great work with your accordion. Well done using index to create unique data-targets